script.js

```diff
@@ -2,49 +2,73 @@
 //Declare a variable called 'name' that has the value of your name.
 
 /*code here*/
-
+var name="Jose";
 //#2
 //create an if statement that checks to see if your name is equal to 'Ben'
 //If yes, log 'yes'
 //otherwise log 'no'
 //note: use console.log to log the string
 
 /*code here*/
-
+if( name==='Ben')
```

Comment:
I'd also recommend putting a space between `if` and the opening `(`.  I'd also remove the spaces before `name` and after `'Ben'` and likewise put spaces around `===`.  `if (name === 'Ben')`